.github/workflows/build-and-test-pr.yaml

```diff
@@ -36,14 +36,6 @@ jobs:
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
           build-depends-repos: build_depends.repos
 
-      - name: Upload coverage to Codecov
```

Comment:
@KeisukeShima Could you apply this to https://github.com/autowarefoundation/autoware_common?

Reply:
Okay. I noticed that I need to send a PR to autoware_common in order to change CI settings.

Reply:
@KeisukeShima I've added it here.
https://github.com/autowarefoundation/autoware_common/pull/14

Reply:
Great! Thank you!

Reply:
Re-runed.
https://github.com/autowarefoundation/autoware.universe/actions/runs/1776519735